platform-api/tests/test_auth.py

```
@@ -0,0 +1,172 @@
+        return self.store.get(key)
+
+    async def put(self, key, value, **kwargs):
+        self.store[key] = value
+
+
+class MockDB:
+    def __init__(self, return_value=None):
+        self.return_value = return_value
+        self.call_count = 0
+
+    def prepare(self, query):
```

Ruff (0.15.12)

[warning] Unused method argument: `query`

(ARG002)